src/parser/paladin/protection/modules/features/ShieldOfTheRighteous.js

```diff
@@ -49,7 +49,7 @@ class ShieldOfTheRighteous extends Analyzer {
   constructor(...args) {
     super(...args);
     const boss = findByBossId(this.owner.boss.id);
-    this._tankbusters = boss.fight.softMitigationChecks;
+    this._tankbusters = [...boss.fight.softMitigationChecks.physical, ...boss.fight.softMitigationChecks.magical];
```

Comment:
this should only have physical. SotR does nothing against magic damage as of 8.0

Reply:
Fixed.